pkg/smokescreen/config.go

```diff
@@ -94,8 +96,10 @@ func NewConfig() *Config {
 		ConnTracker:             new(sync.Map),
 		Log:                     log.New(),
 		Port:                    4750,
-		ExitTimeout:             60 * time.Second,
+		ExitTimeout:             500 * time.Minute,
```

Comment:
500 minutes, or 8.33 hours, is the default amount of time you can delay before the termination cycle is completed on AWS.